### PR TITLE
Show and edit timezone and interview timing preference

### PR DESCRIPTION
**Pull request by bytheway875**

### CODE REVIEW

File: app/views/hub/clients/show.html.erb [hunk 1]
```diff
@@ -11,12 +11,15 @@
 
     <div><%= t(".intake_language") %>: <%= @client.intake&.locale ? t("general.language_options.#{@client.intake&.locale}") : "N/A" %></div>
     <div><%= t(".phone_interview_language") %>: <%= @client.intake&.preferred_interview_language ? t("general.language_options.#{@client.intake&.preferred_interview_language}") : "N/A" %></div>
+    <div><%= "Preferred Interview Time" %>: <%= @client.intake&.interview_timing_preference %></div>
+    <div><%= t("general.timezone") %>: <%= displayed_timezone(@client.intake&.timezone) || "N/A" %></div>
```
Comment (sniemeyer13) [inline app/views/hub/clients/show.html.erb:15]: I18n as well?

File: app/views/hub/clients/edit.html.erb [hunk 2]
```diff
@@ -12,6 +12,8 @@
       <%= f.cfa_input_field(:primary_first_name, t(".legal_first_name"), classes: ["form-width--long"]) %>
       <%= f.cfa_input_field(:primary_last_name, t(".legal_last_name"), classes: ["form-width--long"]) %>
       <%= f.cfa_input_field(:preferred_name, t(".preferred_name"), classes: ["form-width--long"]) %>
+      <%= f.cfa_select(:timezone, t("general.timezone"), timezone_select_options) %>
+      <%= f.cfa_input_field(:interview_timing_preference, "Interview Timing Preference") %>
```
Comment (sniemeyer13) [inline app/views/hub/clients/edit.html.erb:16]: Can we I18n this?

**Review by sniemeyer13 — APPROVED** [review]
Looks good! Alex and I added two small comments about I18n.

**Comment by bytheway875** [comment]
@sartan @sniemeyer13 Good catches!! I updated both parts ("N/A" seems to be N/A in both languages, but using a translation node is better) -- I actually am realizing idk if we have any translations for the timezones themselves, but I'm going to deal with that as part of another ticket.

**Pull request merged**